drivers/pwm/pwm_nrfx.c

```diff
@@ -366,6 +430,11 @@ static int pwm_nrfx_init(const struct device *dev)
 	COND_CODE_1(PWM_HAS_PROP(idx, memory_regions),			      \
 		(DT_PROP_OR(PWM_MEM_REGION(idx), zephyr_memory_attr, 0)), (0))
 
+#define PWM_INIT_PRIORITY(idx)								\
```

Comment:
Please align to solution from SPIM, including the comment:
https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/spi/spi_nrfx_spim.c#L768-L781